### DISCUSSION TITLE
请问苹果笔记本应该下载哪个安装包呢？

### DISCUSSION THREAD

**Discussion by redskunk1231** [qier222/YesPlayMusic#1497] [Q&A] [answered]
请问苹果笔记本应该下载哪个安装包呢？小白问题希望知道的大神给我回答，急！跪谢！

**Comment by liaosky** [comment, marked as answer]
[YesPlayMusic-mac-0.4.4-1-universal.dmg](https://github.com/qier222/YesPlayMusic/releases/download/v0.4.4-1/YesPlayMusic-mac-0.4.4-1-universal.dmg)
